### PR TITLE
Adding menu configuration for A/A experiment and A/B experiment

### PR DESCRIPTION
**Pull request by jhugman**
This is the fenix app part of #11987 and [SDK-41][1].

Not in this: [the assets][1], or [the telemetry][2], or the staging server config.

[1]: https://jira.mozilla.com/browse/SDK-41
[2]: https://jira.mozilla.com/browse/SDK-114

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by jhugman** [comment]
Given when menu items are populated, we're going to be racing to enroll for first time run.

**Comment by jhugman** [comment]
I'd love an asset from @mozilla-mobile/ux , or directions where to find the correct one! HALP PLEASE!

**Comment by violasong** [comment]
Hi @jhugman, are you talking about "star in tray" icon? I originally added it to a Jira issue, but here it is again:
[Bookmarks List.svg.zip](https://github.com/mozilla-mobile/fenix/files/5617319/Bookmarks.List.svg.zip)

**Comment by jhugman** [comment]
> I originally added it to a Jira issue, but here it is again:
> [Bookmarks List.svg.zip](https://github.com/mozilla-mobile/fenix/files/5617319/Bookmarks.List.svg.zip)

Thank you so much!

**Comment by jhugman** [comment]
Hi, @violasong, would you mind providing a square asset? 

The way I'm using it at the moment (via the android studio asset importer), it's coming out as 24dp x 29.805dp.

**Comment by jhugman** [comment]
Current status.

showing new bookmark list icon in light mode showing new bookmark list icon in dark mode

**Comment by jhugman** [comment]
Pending an additional asset, this is ready for review.

File: app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt [hunk 1]
```diff
@@ -96,17 +98,41 @@ class HomeMenu(
...
+            bookmarksIcon,
             primaryTextColor
         ) {
             onItemTapped.invoke(Item.Bookmarks)
         }
 
+        // We want to validate that the Nimbus experiments library is working, from the android UI
+        // all the way back to the data science backend. We're not testing the user's preference
+        // or response, we're end-to-end testing the experiments platform.
+        // So here, we're running multiple identical branches    with the same treatment, and if the
```
Comment (eliserichards) [inline app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt:121]: Nit: whitespace
I really appreciate these comments 👍

Comment (eliserichards) [reply app/src/main/java/org/mozilla/fenix/home/HomeMenu.kt:121]: also it looks like a `let` block not a `let?`

**Review by eliserichards — COMMENTED** [review]
Two small nits. This is very clear and the comments make this very accessible. I'm happy to approve this once it is ready 🥇

**Comment by violasong** [comment]
> Hi, @violasong, would you mind providing a square asset?
> 
> The way I'm using it at the moment (via the android studio asset importer), it's coming out as 24dp x 29.805dp.

Ah, sorry about that! It should actually fill a 24 x 24 px space. I re-exported it with extra spacing so that it's square.
[bookmarks list x24.svg.zip](https://github.com/mozilla-mobile/fenix/files/5626058/bookmarks.list.x24.svg.zip)

Let me know if this looks right when you add it in.

**Comment by jhugman** [comment]
@violasong THANK YOU!
@eliserichards Ready for review. Please land when happy!

**Review by eliserichards — APPROVED** [review]
Wonderful! Let's land this! 🚀

**Pull request merged**